README.md

```diff
@@ -1,6 +1,7 @@
-bouncingBall
-============
+# bouncingBall
 
-Entry-point: MAIN.m
+Entry-point: `MAIN.m`
 
 Runs a simulation of a ball bouncing over hilly terrain. Simulation implemented using ode45 and event detection.
+
+There is also a minimalistic version called `MAIN_minimaistic.m`. It attempts to be a tiny example of the same problem (albeit with significantly limited functionality) all in one file. Note that older versions of Matlab do not support function definitions inside of scripts. `MAIN_minimalistic.m` has, therefore, been defined as a function. `EventFunction_minimalistic` has to be a fully-defined function rather than an anonymous function since it is required to have 3 outputs.
```

Comment:
Thanks for the extra documentation!   FYW: there is actually a way to do multiple returns from a lambda function, but it is not obvious and also hard to read, so the way that you implemented it is good.

You might consider organizing this a bit, for example making a section for each executable, something like:
```markdown
# `MAIN.m`
ToDo:  documentation for main
...
# `MAIN_minimalistic.m`
ToDo:  add your documentation here
...
```